### ISSUE TITLE
Latest build.. still have problems

### ISSUE THREAD

**Issue by pawelomega** [Chips-fr/uae4arm-rpi#47]
Hello M8.

I tryed today your last source. Sounds is totally dis sychronized with video. It's a little better then last your corrects but still far from the previous version before adjustments.

**Comment by drakonia667** [comment]
Yep, i can confirm that. I've back to previous version and sound problems gone.
But there's no way to configure arrows+ctrl as fire in previous version?
I'm using Protracker, in latest version sounds lags awfully.
